docs/pages/web/banneroverlay.js

```diff
@@ -0,0 +1,223 @@
+
+          Icons and thumbnails on BannerOverlay are purely decorative, and can therefore have an empty string as the \`accessibilityLabel\`. The thumbnail (Image) or Icon should supply an alt or accessibilityLabel, respectively, if the Image or Icon supplies extra context or information.`}
+        />
+      </AccessibilitySection>
+
+      <MainSection name="Variants">
+        <MainSection.Subsection
+          description={`Remember to localize text and any string within \`primaryAction\` or \`dismissButton\`, as well as title and message.
+
+<b>Please note</b>: Start aligned text is the primary alignment for our Business products. It will be left-aligned in left-to-right languages and right-aligned in right-to-left languages.`}
+        />
+        <MainSection.Subsection title="Image">
+          <MainSection.Card
+            sandpackExample={<SandpackExample code={image} name="Image" layout="column" />}
+            description="With an image for Pin or Board actions, or the Pinterest logo."
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection title="Avatar">
+          <MainSection.Card
+            sandpackExample={<SandpackExample code={avatar} name="avatar" layout="column" />}
+            description="With an Avatar for Profile or Pinner-related messaging."
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection title="Icon">
+          <MainSection.Card
+            sandpackExample={<SandpackExample code={icon} name="icon" layout="column" />}
+            description="For when an icon is needed to represent content that isn’t a pin or a profile."
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection title="Call to Action">
+          <MainSection.Card
+            sandpackExample={<SandpackExample code={callToAction} name="icon" layout="column" />}
+            description="You can have zero, one or two actions using the props `primaryAction` and `secondaryAction`."
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection title="Dismissed by CTA">
+          <MainSection.Card
+            sandpackExample={<SandpackExample code={ctaDismiss} name="icon" layout="column" />}
+            description={`You can optionally have the Call to Action dismiss the BannerOverlay by not setting the dismissButton prop.
+
+<b>Please note:</b> Make sure to either set the CTA to dismiss the BannerOverlay or set the dismissButton prop. Otherwise, the BannerOverlay will be stuck on the screen.`}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+      <MainSection name="Responsive">
+        <MainSection.Subsection>
+          <MainSection.Card
+            description={`UpsellOverlay is responsive to different devices. Therefore, OverlayUpsell dweb is set to a max width of 900 px to preserve a great usability experience and consistency with other components.
```

Comment:
One more instance, calling it UpsellOverlay

Reply:
Fixed!